src/main/java/org/elasticsearch/example/realm/CustomAuthenticationFailureHandler.java

```diff
@@ -68,10 +68,32 @@ public ElasticsearchSecurityException missingToken(RestRequest request) {
     }
 
     @Override
-    public ElasticsearchSecurityException missingToken(TransportMessage message, String action) {
```

Comment:
looks like you replaced the wrong version of this method, should have replaced the one taking a RestRequest